src/RazorMailMessage.Tests/RazorMailMessage.Tests.csproj

```diff
@@ -10,6 +10,7 @@
     <PackageReference Include="Microsoft.CSharp" Version="4.0.0" />
     <PackageReference Include="Moq" Version="4.0.10827" />
     <PackageReference Include="NUnit" Version="2.6.2" />
+    <PackageReference Include="NUnitTestAdapter" Version="2.3.0" />
```

Comment:
This package reference was added, but the changes included in this pull request don't seem to need it. Is there any commit that was not pushed perhaps?